### PR TITLE
@yuki24 => [Onboarding] Style QA PT.2

### PR DESCRIPTION
**Pull request by kanaabe**
Addresses the "In Progress" cards in: https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=4&projectKey=AR&modal=detail&selectedIssue=AR-69

onboarding-qa

### CODE REVIEW

File: src/Components/Onboarding/Steps/Layout.tsx [hunk 1]
```diff
@@ -25,49 +25,40 @@ const Container = styled.div`
...
   position: fixed;
   bottom: 0px;
   left: 0px;
-`
-
-/* Mobile safari doesn't support for the fixed position property:
- *   https://www.eventbrite.com/engineering/mobile-safari-why/
- **/
-const StickyButtonContainer = styled.div`
-  position: sticky;
```
Comment (yuki24) [inline src/Components/Onboarding/Steps/Layout.tsx]: Have you tested this on different sizes of iOS devices? They are tricky and don't always respect `position: fixed`.

File: src/Components/Title.tsx [hunk 2]
```diff
@@ -22,22 +23,16 @@ const Title: React.SFC<TitleProps> = props => {
...
 }
 
 const StyledTitle = styled(Title)`
   font-size: ${props => titleSizes[props.titleSize]};
   color: ${props => props.color};
   margin: 20px 0;
-  ${fonts.secondary.style}
-
-  ${media.sm`
+  ${fonts.secondary.style} ${media.sm`
```
Comment (yuki24) [inline src/Components/Title.tsx]: I wonder if this is prettier that auto-formats code, but at the same time, I'm not sure what the ideal formatting here is.

cc @orta

Comment (orta) [reply src/Components/Title.tsx]: This is prettier doing it, I'd call it as not perfect, but reasonable

Comment (kanaabe) [reply src/Components/Title.tsx]: Ah in the past I've used `;` to separate lines in these instances. Just updated the PR to reflect that!

File: src/Components/Onboarding/Steps/Layout.tsx (hunk 1 above)
Comment (kanaabe) [reply src/Components/Onboarding/Steps/Layout.tsx]: Not sure what's going on with storybooks on Safari rn (lmknow if you're seeing the same and I'll log an issue to keep track of it). But anyway I reverted back to the other implementation with just style updates so we should be covered. 👍

**Comment by yuki24** [comment]
This is 💯

**Pull request merged**